config/kubernetes-sigs/sig-testing/teams.yaml

```diff
@@ -31,16 +31,16 @@ teams:
     - alejandrox1
     - andrewsykim
     - BenTheElder
-    - spiffxp
     - vladimirvivien
+    - xmcqueen
```

Comment:
This one and the others below in this file are actually subproject owners unrelated to the chair role.
We probably need to review these but I think that's a different PR.

Reply:
@BenTheElder to unblock should we move forward with merging this with a follow up to clean up? or @xmcqueen do you have time to update the PR?

Reply:
The file above has the required change and all of the changes in this file should really be a separate cleanup, this PR alters some of the sig subprojects but not all and I think that was just a find + replace.

I don't think that was intended and I don't think we should merge that way. If Aaron wants to drop from these groups that's a different PR